solr/core/src/test/org/apache/solr/core/TestConfigSetService.java

```diff
@@ -108,16 +110,14 @@ public void testConfigSetServiceOperations() throws IOException {
     assertTrue(configSetService.getConfigMetadata(configName).containsKey("foo"));
 
     List<String> configFiles = configSetService.getAllConfigFiles(configName);
-    assertEquals(
-        configFiles.toString(),
-        "[file1, file2, solrconfig.xml, subdir/, subdir/file3, subdir/file4]");
+    assertEquals(configFiles, List.of(file1, file2, solrConfigXml, subdir, file3, file4));
```

Comment:
Perfect example of how it was so readable before and now... it's for example not evident that the output has slashes and what type.  Test code is not like main/prod code; different stylistic choices emphasizing easy readability above more idealistic decomposition.

Reply:
Yea agreed - I'll leave the list,of but will revert back to actual strings.

Reply:
Fixed in c445c08e2f31b18a8b047051a9cea5ca3f582bf1